components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt

```diff
@@ -459,7 +463,9 @@ internal fun HitResult.getLink(): String = when (this) {
     is HitResult.UNKNOWN -> src
     is HitResult.IMAGE_SRC -> uri
     is HitResult.IMAGE ->
-        if (title.isNullOrBlank()) src else title.toString()
+        if (title.isNullOrBlank()) {
+            if (src.length > MAX_TITLE_LENGTH) "image" else src
+        } else title.toString()
```

Comment:
nit: Alternatively, instead of the nested if-else, `take` might work.
```suggestion
        if (title.isNullOrBlank()) src.take(MAX_TITLE_LENGTH) else title.toString()
```

Reply:
IMO we should show the generic "image" title, instead of a substring of the encoded image. What do you think about using this new extension?